operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/informer/InformerConfiguration.java

```diff
@@ -202,12 +213,17 @@ public InformerConfigurationBuilder<R> withGenericFilter(GenericFilter<R> generi
       return this;
     }
 
+    public InformerConfigurationBuilder<R> withCachePruneFunction(UnaryOperator<R> itemStore) {
```

Comment:
Why is the parameter called itemStore?

Reply:
will rename it, before the whole item store was customizable, later removed it

Reply:
fixed